Sources/Plasma/PubUtilLib/plAudio/plAudioSystem_Private.h

```diff
@@ -48,6 +48,10 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include <alc.h>
 #ifdef USE_EFX
 #   include <efx.h>
+#else
+#if __APPLE__
+#   include <OpenAL/OpenAL.h>
```

Comment:
Is this necessary? I'm pretty sure I've been able to compile stuff on macOS with the normal CMake FindOpenAL module without needing to add this explicitly...

In practice, we prefer openal-soft over the system OpenAL because it includes EFX support and isn't deprecated

Reply:
Yeah - in practice I've found Apple's OpenGL fairly unreliable with audio glitches. A lot of other developers are seeing the same issues. I'd be fine moving over to openal-soft entirely and not supporting Apple's OpenAL.